qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -183,7 +183,45 @@ def _get_version(self):
             mes = self.ask(bytes([3, 4]))
             ver = mes[2]
             return ver
-
+    
+    def linsweep(self,start,end,points):
```

Comment:
Please use spaces and type annotations :)
```suggestion
    def linsweep(self, start: float, end: float, points: int) -> List[float]:
```